test/integration/integration_test.go

```diff
@@ -160,6 +160,8 @@ func FeatureContext(s *godog.Suite) {
 	// steps to execute `minishift` commands
 	s.Step(`^Minishift (?:has|should have) state "(Does Not Exist|Running|Stopped)"$`,
 		minishift.shouldHaveState)
+	s.Step(`^Profile (.*) of Minishift (?:has|should have) state "(Does Not Exist|Running|Stopped)"$`,
```

Comment:
I am not sure about the `of Minishift`, could it be:
```
Profile (.*) (?:has|should have) state "(Does Not Exist|Running|Stopped)"
```
?

Also first matching group should be in brackets I think to outline the name out of the sentence. I am wondering how is it that it works for you when in feature.file there is a step: `Given Profile "minishift" of Minishift has state "Does Not Exist"` which uses brackets.

Reply:
> I am not sure about the ```of Minishift```, could it be:
```Profile (.*) (?:has|should have) state "(Does Not Exist|Running|Stopped)"```
?

Not bad. Profile itself describe it as the instance. Would be fine with  ```Profile (.*) (?:has|should have) state "(Does Not Exist|Running|Stopped)"```. 

> Also first matching group should be in brackets I think to outline the name out of the sentence. I am wondering how is it that it works for you when in feature.file there is a step: Given Profile "minishift" of Minishift has state "Does Not Exist" which uses brackets

Not sure what you mean here